core/client/client.go

```diff
@@ -74,6 +69,19 @@ var (
 	Vlog = log.New(ioutil.Discard, "", 0)
 )
 
+// Verifier is used to verify specific outputs from Key Transparency.
+type Verifier interface {
+	// Index computes the index from a VRF proof.
```

Comment:
Index computes the index of what?  Add to the comment to make it really clear :)

Reply:
Index computes the index of an appID, userID pair from a VRF proof, obtained from the server.